packages/ember-page-layout/package.json

```diff
@@ -56,11 +62,28 @@
   "ember-addon": {
     "version": 2,
     "type": "addon",
-    "main": "addon-main.js"
+    "main": "addon-main.js",
+    "app-js": {
+      "./components/page/footer.js": "./dist/_app_/components/page/footer.js",
+      "./components/page/footer/container.js": "./dist/_app_/components/page/footer/container.js",
+      "./components/page/header.js": "./dist/_app_/components/page/header.js",
+      "./components/page/header/container.js": "./dist/_app_/components/page/header/container.js",
+      "./components/page/layout.js": "./dist/_app_/components/page/layout.js",
+      "./components/page/sidebar.js": "./dist/_app_/components/page/sidebar.js",
+      "./components/page/sidebar/container.js": "./dist/_app_/components/page/sidebar/container.js",
+      "./index.js": "./dist/_app_/index.js",
+      "./services/layout.js": "./dist/_app_/services/layout.js",
+      "./utils/guard.js": "./dist/_app_/utils/guard.js",
+      "./initializerDefineProperty-bbf9f274.js": "./dist/_app_/initializerDefineProperty-bbf9f274.js",
+      "./applyDecoratedDescriptor-ff18b166.js": "./dist/_app_/applyDecoratedDescriptor-ff18b166.js"
+    }
```

Comment:
so is how you have to export stuff from V2 addons to make them available to the app now, instead of re-exporting from an `app/*` dir?